easybuild/main.py

```diff
@@ -255,6 +255,10 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     config.init(options, config_options_dict)
     config.init_build_options(build_options=build_options, cmdline_options=options)
 
+    if options.fetch:
+        _log.debug("Fetching sources only, implies stop=fetch, moduletool=NoModulesTool, ignore-osdep.")
+        modtool = NoModulesTool(testing=testing)
```

Comment:
@MisterFruits Why not enhance the `modules_tool` function to handle this, that way you don't need to touch `main.py` at all?

Seems like you just need to return a `NoModulesTool` instance when `get_modules_tool` returns `None` in `modules_tool`?

That's actually better than what it does now, since now it will return `None` when `get_modules_tool` returns `None` somehow, which would make `main.py` go down hard as soon as `modtool` is used for anything...